express/properties/workflow.py

```diff
@@ -200,6 +203,7 @@ def get_workflow_specific_config(self) -> dict:
         specific_config = {
             "units": train_subworkflow_units,
             "subworkflows": predict_subworkflows,
+            "isUsingDataset": self.is_using_dataset(),
```

Comment:
Nit, and no need to do the change, but we could use a property decorator for this and avoid calling the function

Reply:
Noted - I'll merge this in for now per https://exabyte.atlassian.net/browse/SOF-4391?focusedCommentId=41338, but I'll add the property decorator in the cleanup work being done in 4399